python/lsst/ts/aos/analysis/blockUtils.py

```diff
@@ -0,0 +1,101 @@
+# This file is part of ts_aos_analysis.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ['add_property', 'build_configuration_schema']
+
+def add_property(prop_name: str, prop_details: dict, indent_level: int = 1) -> str:
+    """ Add a property to a configuration schema string.
```

Comment:
It's supposed to be a json-formatted string right? Would be helpful to mention that.